Binary Search Trees/Binary Search Tree Implementation Complete.py

```diff
@@ -0,0 +1,202 @@
+class Node:
+    def __init__(self,value):
+        self.value = value
+        self.left = None
+        self.right = None
+    
+    def insert(self, value):
+        if self.value == value:
+            return
+        if self.value > value:
+            if self.left:
+                return self.left.insert(value)
+            else:
+                self.left = Node(value)
+        if self.value < value:
+            if self.right:
+                return self.right.insert(value)
+            else:
+                self.right = Node(value)
+    
+    def find(self, value):
+        found = False
+        if self.value == value: #check to see if the current node's value matches the search value. NOTE - initiation self == root.
```

Comment:
Better to put long comments in new lines. You have to scroll a lot to the right to read this comment.

Reply:
Okok